src/components/about/About.component.tsx

```diff
@@ -0,0 +1,37 @@
+import styles from './about.module.css';
+
+export const AboutComponent = () => {
+    const aboutDescriptionClass = styles['about-description'];
```

Comment:
Good job that you are trying to reuse your code. But in general it will be much better to create another component e.g `Typography`, apply styling directly to it and reuse it in the `About` component

Reply:
I can't handle this task for now 😢